contest/static/contest/css/contestArchive.css

```diff
@@ -17,10 +17,6 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 */
-
-#contestArchive.container {
-  width: 90%;
-}
 .black {
   color: black;
```

Comment:
``.black`` and ``.red`` in no longer used in his codes. As a result, the whole file can be deleted.